oscar_stripe_sca/loading.py

```diff
@@ -0,0 +1,22 @@
+import importlib
+import logging
+
+
+def get_class_by_path(class_path):
```

Comment:
Why not use Oscar's built-in `oscar.core.loading.get_class`? https://django-oscar.readthedocs.io/en/latest/ref/core.html#oscar.core.loading.get_class

Or Django's built-in `django.utils.module_loading.import_string`? https://docs.djangoproject.com/en/5.2/ref/utils/#django.utils.module_loading.import_string